recipes/openssl/3.x.x/conanfile.py

```diff
@@ -470,6 +467,11 @@ def _create_targets(self, cflags, cxxflags, defines, ldflags):
             cflags.append("-fembed-bitcode")
             cxxflags.append("-fembed-bitcode")
 
+        if is_msvc(self) or self._is_clang_cl:
```

Comment:
AutotoolsToolchain currently does:
```
    def _msvc_extra_flags(self):
        if is_msvc(self._conanfile) and check_min_vs(self._conanfile, "180",
                                                     raise_invalid=False):
            return ["-FS"]
        return []
```
So not covering ClangCL, maybe it needs to be added there?

Reply:
oh this block needs to be removed 100% 
`-FS` is already passed by `AutotoolsToolchain` - so this is, at best, redundant

the problem is that ... is still fails sporadically with the concurrent access error, despite the fact that -FS is passed. To be fair jom was just for faster local testing, but it would be good if this was working well

Reply:
These can be dropped, but it's only available since `2.1.0`, so you might want to update `required_conan_version` accordingly.